app.js

```diff
@@ -47,28 +56,17 @@ CookieStore.prototype.makeTheTable = function(tableToDrawInto){
   tableToDrawInto.appendChild(hourlySalesRowEl);
 };
 
-var tableEl = document.createElement('table');
-console.log(tableEl);
-var sectionEl = document.getElementById('cookieSalesTable');
-sectionEl.appendChild(tableEl);
-
-var hoursRowEl = document.createElement('tr');
-var hoursHeaderEl = document.createElement('th');
-hoursRowEl.appendChild(hoursHeaderEl);
-for(var tableHours = 0; tableHours < storeHours.length; tableHours++){
-  var hoursDataEl = document.createElement('td');
-  hoursDataEl.textContent = storeHours[tableHours];
-  hoursRowEl.appendChild(hoursDataEl);
+function makeHoursRow(){
+  for(var tableHours = 0; tableHours < storeHours.length; tableHours++){
+    var hoursDataEl = document.createElement('td');
+    hoursDataEl.textContent = storeHours[tableHours];
+    hoursRowEl.appendChild(hoursDataEl);
+  }
+  tableEl.appendChild(hoursRowEl);
 }
-tableEl.appendChild(hoursRowEl);
-
-var stores = [firstAndPike, seatacAirport, seattleCenter, capitolHill, alki];
 
 console.log('==============EVENT LISTENERS===============');
```

Comment:
This tells you when you're about to attach the event listener, not when the event listener fires. Where could you put this to make it more informative?